tests/integration/test_logcollector/test_location/test_location.py

```diff
@@ -0,0 +1,182 @@
+     'file_type': 'wildcard_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'duplicated', 'duplicated.txt'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'duplicated', 'duplicated.txt')],
+     'log_format': 'syslog', 'path_2': os.path.join(temp_dir, 'wazuh-testing', 'duplicated', 'duplicated.txt'),
+     'file_type': 'duplicated_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'file.log-%Y-%m-%d'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing',f'file.log-{date}')], 'log_format': 'syslog',
+     'file_type': 'single_file'},
+    {'location': os.path.join(temp_dir, 'wazuh-testing', 'multiple-logs', '*'),
+     'files': [os.path.join(temp_dir, 'wazuh-testing', 'multiple-logs', 'multiple')],
+     'log_format': 'syslog', 'file_type': 'multiple_logs'}
+]
+
+if sys.platform != 'win32':
+    for case in metadata:
+        if case['location'] == os.path.join(temp_dir, 'wazuh-testing', '*'):
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', 'テスト.txt'))
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', 'ИСПЫТАНИЕ.txt'))
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', '测试.txt'))
+            case['files'].append(os.path.join(temp_dir, 'wazuh-testing', 'اختبار.txt'))
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['LOCATION']}_{x['LOG_FORMAT']}" for x in parameters]
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+
+# Fixtures
+@pytest.fixture(scope="module")
+def create_directory():
+    """Create expected directories."""
+    os.makedirs(os.path.join(temp_dir, 'wazuh-testing', 'multiple-logs'), exist_ok=True)
+    os.makedirs(os.path.join(temp_dir, 'wazuh-testing', 'depth1', 'depth2'), exist_ok=True)
+    os.makedirs(os.path.join(temp_dir, 'wazuh-testing', 'duplicated'), exist_ok=True)
+    yield
+    rmtree(os.path.join(temp_dir, 'wazuh-testing'), ignore_errors=True)
+
+
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def get_local_internal_options():
+    """Get configurations from the module."""
+    return local_internal_options
+
+
+@pytest.fixture(scope='module')
+def create_files(request, get_configuration):
+    """Create expected files."""
+    files = get_configuration['metadata']['files']
+    file_type = get_configuration['metadata']['file_type']
+
+    if file_type != 'non_existent_file':
+        for file_location in files:
+            if file_type == 'multiple_logs':
+                for i in range(2000):
+                    open(f'{file_location}{i}.txt', 'w').close()
+            else:
+                open(file_location, 'w').close()
+    yield
+

```

Comment:
```suggestion
            else:
                open(file_location, 'w').close()
                
    yield


```

Reply:
Fixed in 675fe8071d37aabfec50859993e777238e0f3f45